### PR TITLE
Fix a bug that toast window is shown below keyboard in iOS 11

### LINKED ISSUE #117: Toast appears behind keyboard on iOS 11
https://drive.google.com/file/d/0Bys2UYWmcgc0T1l5cDJZcDJDejA/view

### PR DESCRIPTION
**Pull request by devxoul**
It fixes #117

### CODE REVIEW

**Comment by yuan-java** [comment]
The fix will cause my app crash when clicking back button on the navigation bar as long as the keyboard appears.

**Comment by devxoul** [comment]
@yuan-java, could you please test it again? I changed the implementation.

**Comment by RafaelPlantard** [comment]
Can I test too @devxoul?

**Comment by devxoul** [comment]
@RafaelPlantard sure thing!

**Comment by Coeur** [comment]
Sounds good. I made small suggestions for you at #138.
Also, I'm not sure that we still need `bringWindowToTop`: it's now rendered useless and can be deleted.

File: Sources/ToastWindow.swift [hunk 1]
```diff
@@ -136,6 +154,21 @@ open class ToastWindow: UIWindow {
     }
   }
 
+  @objc func keyboardWillShow() {
+    self.bringWindowToTop()
```
Comment (Coeur) [inline Sources/ToastWindow.swift]: Not needed: I would remove that line.

**Comment by Coeur** [comment]
To be honest, I forgot about that pull request. :-D

**Comment by Coeur** [comment]
@devxoul I've merged it. But if we get a report of a rejection by Apple for adding subviews to the keyboard, we'll have to revert it.

**Comment by Coeur** [comment]
@devxoul let's do a release with that change, to get feedback on it.

**Comment by dhruvalpatel18** [comment]
Still doesn't work for me. iOS 12+

**Pull request merged**